### PR TITLE
draft of interview finished

### PR DESCRIPTION
**Pull request by jcolomb**

### CODE REVIEW

File: content/blog/2022-10-06-interview-white-rabbit/index.en.md [hunk 1]
```diff
@@ -0,0 +1,326 @@
+---
+title: 'Interview: White rabbit'
+author: 
+  - Open make team
```
Comment (jcolomb) [inline content/blog/2022-10-06-interview-white-rabbit/index.en.md:4]:
```suggestion
  - Open make interview team
```

File: content/blog/2022-10-06-interview-white-rabbit/index.en.md [hunk 2]
```diff
@@ -0,0 +1,326 @@
...
+- [The project](#the-project)
+- [The Hardware](#the-hardware)
+- [The Research outputs](#research-outputs)
+- [The participants](#participants)
+
+
+>Interviewee: Javier Serrano (CERN) & Amanda Diez Fernandez (CERN)
+>
+>Interviewers: Robert Mies (TU Berlin) & Moritz Maxeiner (FU Berlin)
+
```
Comment (jcolomb) [inline content/blog/2022-10-06-interview-white-rabbit/index.en.md:28]:
```suggestion
Transcription and editing: Diana Americano Guerrero, Julien Colomb
```

**Review by jcolomb — COMMENTED** [review]
modify authors

**Comment by MIE5R0** [comment]
As an author of this blog post, I agree for its publication.

File: content/blog/2022-10-06-interview-white-rabbit/index.en.md [hunk 3]
```diff
@@ -0,0 +1,326 @@
...
+At CERN, the European Laboratory for Particle Physics, researchers perform experiments in which particle beams are passed through a series of increasingly higher energy accelerators, until they reach the LHC, the Large Hadron Collider, which is currently the biggest accelerator at CERN. There, the beams are made to collide so the physicists can study the outcome.
+
+From an engineering perspective, this requires a distributed real-time system to ensure all components are synchronized at sub-nanosecond accuracy - something previously existing solutions were unable to provide. The project to develop the necessary hard- and software as open source, spearheaded at CERN by a team of electronics designers and Linux kernel developers around Javier Serrano (section leader), became known as “White Rabbit”. Amanda Diez has joined the project recently to increase internal and external communitation efforts.
+
+{{< card  "The White Rabbit in a nutshell">}}
+
+![image alt](https://ohwr.org/project/wr-switch-hw/uploads/8bec64c2bc708e20d516d4ddb9ce8da4/wrs3_W02-v3.4-smc.jpg "WR Switch v3.4: Front view, FMC connectors") 
+*WR Switch v3.4: Front view, FMC connectors, shared under [no license?]*
```
Comment (MoritzMaxeiner) [inline content/blog/2022-10-06-interview-white-rabbit/index.en.md:41]: Circuit board and relevant mechanics are licensed under `CERN OHL v.1.2`.

See https://ohwr.org/project/wr-switch-hw/tree/master/circuit_board/wrs_v3 and https://ohwr.org/project/wr-switch-hw/blob/master/mechanics/v3.4/pdf/01%20-%20Front%20and%20top%20view.pdf

Comment (MoritzMaxeiner) [reply content/blog/2022-10-06-interview-white-rabbit/index.en.md:41]: If you mean under which license the image is shared under, I would write `, by Grzegorz Daniluk and Erik van der Bij`, as they are the ones listed at the bottom of the documentation page where the images are: https://ohwr.org/project/wr-switch-hw/wikis/home

File: content/blog/2022-10-06-interview-white-rabbit/index.en.md [hunk 4]
```diff
@@ -0,0 +1,326 @@
...
+{{< /card2 >}}
+>How is it funded, overall?
+ 
+ {{< expand "Show answer">}}
+A lot of the funding comes from CERN. Traditionally, before we did open hardware, these types of ad-hoc systems for the accelerators were designed at CERN and paid completely by CERN. So, we had a budget for this project. A lot of the development of the core infrastructure for White Rabbit happened thanks to this budget. One of the differences with respect to past efforts at CERN was that we involved industry from day one, and there was a grant from the Spanish government. I think it was called industry for science. It was supposed to give Spanish companies a competitive advantage for these types of projects, collaborating with big international laboratories to get some kind of competitive edge. So, we benefited from that grant. Two Spanish companies worked on the initial design of White Rabbit, one on the hardware side and one on the software side. And that was a very good initial push. Other laboratories like GSI in the south of Germany [GSI Helmholtz Centre for Heavy Ion Research] joined very early on and they also contributed financially and in-kind with the effort of their designers.
```
Comment (MoritzMaxeiner) [inline content/blog/2022-10-06-interview-white-rabbit/index.en.md:114]: Turn GSI in Brackets into a link? https://www.gsi.de/en/start/news

Comment (MoritzMaxeiner) [reply content/blog/2022-10-06-interview-white-rabbit/index.en.md:114]: Same with Nikhef: https://www.nikhef.nl/

**Review by MoritzMaxeiner — APPROVED** [review]
I would change the minor things listed as comments, but:

As an author of this blog post, I agree for its publication.

**Review by MoritzMaxeiner — APPROVED** [review]

**Comment by j-sp** [comment]
As a co-author of this blog post, I agree with its publication.
Javier Serrano

**Comment by adiezfer** [comment]
As a co-author of this blog post, I agree with its publication

**Pull request merged**